src/encoded/tests/utils.py

```diff
@@ -21,33 +23,114 @@ def post_item(
 
 
 def patch_item(
-    testapp: TestApp, patch_body: Dict[str, Any], collection: str, status: int = 200
+    testapp: TestApp, patch_body: Dict[str, Any], identifier: str, status: int = 200
 ) -> Dict[str, Any]:
-    resource_path = get_formatted_resource_path(collection)
+    """PATCH content to given item."""
+    resource_path = get_formatted_resource_path(identifier)
     response = testapp.patch_json(resource_path, patch_body, status=status)
     if status == 200:
         return response.json["@graph"][0]
     return response.json
 
 
 def get_item(
-    testapp: TestApp, identifier: str, frame: Optional[str] = None
+        testapp: TestApp,
+        identifier: str,
+        collection: Optional[str] = None,
+        frame: Optional[str] = None,
+        status: Optional[int] = None,
 ) -> Dict[str, Any]:
-    resource_path = get_formatted_resource_path(identifier, add_on=f"frame={frame}")
-    response = testapp.get(resource_path)
+    """Get item view with given frame, following redirects."""
+    add_on = get_frame_add_on(frame)
+    resource_path = get_formatted_resource_path(
+        identifier, collection=collection, add_on=add_on
+    )
+    response = testapp.get(resource_path, status=status)
```

Comment:
Consider adding `datastore` argument as well for combined ES/Postgres tests that want to request `datastore=database`